settings.gradle

```diff
@@ -3,4 +3,5 @@ rootProject.name = 'claim-store'
 include 'email-client'
 include 'cmc-domain-model'
 include 'cmc-domain-sample-data'
+include 'cmc-ccd-adapter'
```

Comment:
I think all these modules should have cmc removed from them,
domain-model
domain-sample-data
ccd-adaptor

Reply:
Will do in a separate PR.